account_statement_import_base/views/account_bank_statement_line.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" ?>
+<!--
+  Copyright 2022 Akretion France (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  Licence LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0).
+-->
+<odoo>
+
+<record id="view_bank_statement_line_form" model="ir.ui.view">
+    <field name="model">account.bank.statement.line</field>
+    <field name="inherit_id" ref="account.view_bank_statement_line_form" />
+    <field name="arch" type="xml">
+        <field name="statement_id" position="attributes">
+            <attribute
+                    name="invisible"
+                >not context.get('statement_line_main_view')</attribute>
+        </field>
+        <xpath expr="//field[@name='company_id']/.." position="attributes">
+            <attribute name="col">2</attribute>
+        </xpath>
+        <field name="sequence" position="attributes">
+            <attribute name="invisible">1</attribute>
+        </field>
+        <!--
+            I don't see anything interesting in the 'ref' field of the
+            move linked to the statement line
+            and the field move_id is displayed in the view.
+            But if you think it's useful to see this field,
+            don't hesitate to remove the 2 lines below
+        -->
+        <field name="ref" position="attributes">
```

Comment:
Please don't hide fields in standard modules. Let it goes your

Reply:
100% don't do that

Reply:
ok, I removed the XML instruction for hiding "ref" in statement line form view